salt/metalk8s/roles/etcd/running.sls

```diff
@@ -0,0 +1,3 @@
+# creates the manifest YAML and wait for the member to be up and running
+include:
+  - metalk8s.kubernetes.etcd
```

Comment:
This state really needed ?

For me it's not needed and we shouldn't create a state like that except if we want somewhere to run only this state and not all the `roles.etcd` (which is not the case for the moment)